client/blocks/continue-as-user/index.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function ContinueAsUser( { userName, redirectUrl } ) {
+	const translate = useTranslate();
+
+	if ( ! userName || ! redirectUrl ) {
+		return null;
+	}
+
+	const redirectLink = <a href={ redirectUrl }>{ userName }</a>;
+
+	return (
+		<div className="continue-as-user">
+			{ translate( 'or continue as {{redirectLink/}}', {
+				components: { redirectLink },
+				context:
```

Comment:
I've been told by @lsl that `comment` is the param you want to use here. `context` is meant to be used to distinguish different meanings of the same string. E.g., "open" as verb vs. "open" as adjective.

Changing this will make the i18n bot happy, too 🙂

Reply:
Done 🙂 Thanks!